web/i18n/translater.js

```diff
@@ -0,0 +1,25 @@
+const i18n = {};
+
+function getUserLanguage() {
+  console.info(`[Twake Mail] Current Language: `, navigator.language);
+  return navigator.language.split('-')[0];
```

Comment:
Need check for case when `navigator.language` is null || 'undefine' || null so return default language